pages/about.jsx

```diff
@@ -4,23 +4,23 @@ const About = () => {
   return (
     <div>
       <Link to="/" className="start">
-        Home
+        Startseite
       </Link>
 
       <img className="logo_about" src="../src/images/Logo.png" alt="" />
 
       <div className="about_hawify">
         <p>
-          <strong>HAWI.FY</strong> ist eine Kombination aus den Worten "Hauswirtschaft" und dem
-          englischen Wort "Identify" (deu.: "identifizieren"). Die Anwendung
-          wurde im Rahmen des Web-Development Bootcamps von "neuefische"
-          programmiert. Die Programmierung dieser Anwendung betrug insgesamt 
-         vier Wochen und dient als nützliche Hilfestellung für den
-          Haushaltsalltag. Die primären Komponenten bestehen aus einem
-          Kassenbuch und einer Einkaufsliste. Als weitere nützliche Features
-          wurden noch ein Taschenrechner mit einfacher Funktion und ein Kalender
-          für die täglichen "To-Do's" einprogramiert. Erweiterungen der
-          Anwendung sind geplant.
+          <strong>HAWI.FY</strong> ist eine Kombination aus den Worten
```

Comment:
If you use the `strong` to only display it in bold, then I would recommend using a `span` and style it accordingly. The `strong` elements semantic meaning is about conveying an importance. You can read about it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong